Tests/Web/html/tests/usersystem_tests_helpers.js

```diff
@@ -48,13 +48,15 @@ export async function logOut(userSystem) {
  * @param {!boolean} [pushCleanup] 
  * @returns {Promise<?string>} the userId of the logged in account
  */
-export async function logIn(userSystem, email = DEFAULT_LOGIN_EMAIL, password = DEFAULT_LOGIN_PASSWORD, ageVerified = true, expectedResult = Services.EResultCode.Success, pushCleanup = true) {
+export async function logIn(userSystem, email = DEFAULT_LOGIN_EMAIL, password = DEFAULT_LOGIN_PASSWORD, ageVerified = true, expectedResult = Services.EResultCode.Success,expectedFailureResultCode = Services.EResultBaseFailureReason.None, pushCleanup = true) {
```

Comment:
And here with spaces

Reply:
done

Reply:
I can't see the changes. There should be a space here: `EResultCode.Success,expectedFailureResultCode`

Reply:
sorted